gradle/dependencies.gradle

```diff
@@ -0,0 +1,97 @@
+def groovyVer = "2.5.11"
```

Comment:
Do we need this file and the `java.gradle`? 

in OTEL repo these are reused for other modules. since we have only smoke tests module we could copy there only what is needed. For instance we define versions in our root build script.

Reply:
@pavolloffay  have removed dependencies.gradle but not java.gradle.